### DISCUSSION TITLE
Move picture up down in full screen

### DISCUSSION THREAD

**Discussion by OleRHH** [smplayer-dev/smplayer#376] [Q&A] [answered]
SMPlayer Version 21.10.2 (Qt5.15.2, MPV 0.33.1)
Linux kubuntu 21.10

Hi,

When viewing 21/9 videos, I would like to move the image to the bottom edge (so that there is only one black edge at the top).
The key combinations ALT+DOWN (and all other directions) only work if I have previously zoomed into the video.

Is there any possibility?

I use a projector and it is more convenient not to have to look up so much, and I have a screen masking for the top area.

Thank you very much, I really love this player for years.

**Comment by smplayer-dev** [comment, marked as answer]
If the screen is 16:9, add this to preferences -> advanced -> mplayer/mpv -> video filters:
`pad=w=iw:h=iw/16*9:y=oh-ih`

(filter for mpv only)

**Reply by OleRHH** [reply]
Thank you very much for your quick reply.

Yes, it is a 16:9 screen. I tried your solution but it had no effect. I renamed ~/.config/smplayer to create a new profile in case I did something wrong before. But that didn't help. 
Playing around, I discovered the option to "Run MPlayer/mpv in its own window". When enabled, I can move the video up/down in the MPV window. This is an acceptable solution, but now I'm stuck changing the step size, which is way too high. I tried using an input.conf file that  works well when using mpv as standalone. I have read that this file is ignored when using smplayer. Is there another way to change the step size?

**Reply by smplayer-dev** [reply]
What do you mean with "step size"?

This version shouldn't ignore the mpv config files:
https://github.com/smplayer-dev/smplayer/releases/tag/v21.10.0.10046

**Reply by OleRHH** [reply]
Perfect, it works now!

What I meant to say was that one keystroke was not enough to move the video to the bottom of the screen, but the second one already moved the video way out of it.

I now have these two custom keyboard shortcuts (the default value was to high for me):
Alt+up    add video-pan-y  0.01         # default 0.1
Alt+down  add video-pan-y -0.01      # default 0.1

Thank you very much!

**Reply by reisa211** [reply]
THE REAL SOLUTION!

1. Change the option "allow_video_movement" to true in smplayer.ini (Help --> Open configuration folder)
2. Use Alt + arrows UP/DOWN 
3. GG ;PPPpppp

The problem here is when you move the picture up or down the subs move with the picture (not like MPC-HC) and if u set it -1, 2, 3 etc. the subs hides. How to solve this? I want subtitles to be at the very bottom of the screen and moving the picture up and down like MPC-HC! Thank you!
